app/src/lib/stores/git-store.ts

```diff
@@ -1647,18 +1648,15 @@ export class GitStore extends BaseStore {
 
   public async pruneForkedRemotes(openPRs: ReadonlyArray<PullRequest>) {
     const remotes = await getRemotes(this.repository)
-    const prRemotes = new Set<string>()
 
-    for (const pr of openPRs) {
-      if (pr.head.gitHubRepository.cloneURL !== null) {
-        prRemotes.add(pr.head.gitHubRepository.cloneURL)
-      }
-    }
+    const remotesToPrune = findForkedRemotesToPrune(
+      remotes,
+      openPRs,
+      this.allBranches
+    )
 
-    for (const r of remotes) {
-      if (r.name.startsWith(ForkedRemotePrefix) && !prRemotes.has(r.url)) {
-        await removeRemote(this.repository, r.name)
-      }
+    for (const r of remotesToPrune) {
+      await removeRemote(this.repository, r.name)
```

Comment:
Here on the other hand I think we can afford to be a bit more verbose. We typically use 1 char variables in anonymous functions but not so much in 
```suggestion
    for (const remote of remotesToPrune) {
      await removeRemote(this.repository, remote.name)
```